src/views/MissionPlanningView.vue

```diff
@@ -2126,35 +2182,65 @@ const vehiclePosition = computed((): [number, number] | undefined =>
     : undefined
 )
 
+// Create marker for the vehicle
 const vehicleMarker = ref<L.Marker>()
-watch(vehicleStore.coordinates, (coords, prevCoords) => {
-  if (!planningMap.value || vehiclePosition.value === undefined) return
+watch(vehicleStore.coordinates, () => {
+  if (!planningMap.value || !vehiclePosition.value) return
```

Comment:
This code had just been modified 2 commits before, and is being reverted back to its original state here.